tools/build_pytorch_libs.bat

```diff
@@ -202,7 +202,7 @@ goto:eof
                   -DCMAKE_INSTALL_PREFIX="%INSTALL_DIR%" ^
                   -DCMAKE_EXPORT_COMPILE_COMMANDS=1 ^
                   -DCMAKE_C_FLAGS="%USER_CFLAGS%" ^
-                  -DCMAKE_CXX_FLAGS="/EHa /MT %USER_CFLAGS%" ^
+                  -DCMAKE_CXX_FLAGS="/EHa /MP %USER_CFLAGS%" ^
```

Comment:
/MP should be called only when MSVC is used. Otherwise, there'll be too many processes.